### ISSUE TITLE
adding documentation for pyuidemo.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5637]
|<img alt="laurauva's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| laurauva reported|
|-|-|
|Trac ID|trac#5637|
|Type|enhancement|
|Created|2012-04-11 02:57:27Z|

add documentation to pyuidemo.py

Attachments:

* [DiffPyuiDemo](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/cb8/cb8d3994328def255e43214126470bde9a8dea3c/c481dbdd9667a67dcf2fe1216b66891de1bf1dcf) (495 bytes) - added by laurauva on 2012-04-11 17:42:44Z - Diff: pyuidemo.py

<details><summary>Searchable metadata</summary>

```
trac-id__5637 5637
type__enhancement enhancement
reporter__laurauva laurauva
priority__normal normal
milestone__ 
branch__ 
branch_author__laurauva laurauva
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1334113047000000 1334113047000000
changetime__1334363831000000 1334363831000000
version__None None
owner__itamar itamar
cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess set owner to laurauva|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="laurauva's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|laurauva removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_3"></a>|@itamarst set owner to @itamarst|
|-|-|

Other than missing a news file (see http://twistedmatrix.com/trac/wiki/ReviewProcess#Newsfiles, in this case an empty file `twisted/topfiles/5637.misc` since it's a trivial change) looks good.

I'll merge this in the evening, hopefully.

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_4"></a>|@itamarst commented|
|-|-|

Oh, and, for future patches try to run "svn diff" at the root of the Twisted checkout.

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_6"></a>|@itamarst set status to `closed`|
|-|-|

(In [34213]) Merge patch: better docstring for pyuidemo.py

Author: laurauva
Review: itamar
Fixes: [#5637](https://github.com/twisted/twisted/issues/5637)
